src/bin/pg_autoctl/primary_standby.c

```diff
@@ -764,7 +764,7 @@ primary_rewind_to_standby(LocalPostgresServer *postgres)
 	log_info("Rewinding PostgreSQL to follow new primary %s:%d",
 			 primaryNode->host, primaryNode->port);
 
-	if (!pg_ctl_stop(pgSetup->pg_ctl, pgSetup->pgdata))
```

Comment:
For a separate PR, but because of this I noticed that there are some more `pg_ctl_stop` calls. I think a few of these should be changed as well.